ansible/ad_ping.yml

```diff
@@ -0,0 +1,9 @@
+---
+- hosts: all
+  gather_facts: no
+  vars:
```

Comment:
These variables may not be needed because in `tasks/tasks.py` we are setting the variable in the inventory file (in method `cfg_for_windows`). However, if we don't add the variables there this config becomes important. Without it, you will see a bunch of weird errors related to SSL errors and some timeouts.